### PR TITLE
change link expiration message to 60 mins

### PR DESCRIPTION
**Pull request by pvighi**
## What does this change?
Change the pages and email templates to say link expiration is 60 minutes instead of 30

### CODE REVIEW

**Comment by pvighi** [comment]
[update link expiration in gateway and email templates](https://trello.com/c/v4HZGSrN/4079-update-link-expiration-in-gateway-and-email-templates)

File: src/client/pages/EmailSent.tsx [hunk 1]
```diff
@@ -56,7 +56,7 @@ export const EmailSent = ({
       )}
       <MainBodyText>Please follow the instructions in this email.</MainBodyText>
       <MainBodyText>
-        <b>The link is only valid for 30 minutes.</b>
+        <b>The link is only valid for 60 minutes.</b>
```
Comment (pvighi) [inline src/client/pages/EmailSent.tsx:59]: this seems very repetitive. Maybe in the future we can extract this to some common location, either the expiration description or the amount of minutes (if we are sure we are never going to express it in hours or days)

File: cypress/integration/ete/registration/register.cy.ts [hunk 2]
```diff
@@ -209,7 +209,7 @@ describe('Registration flow', () => {
         expect(body).to.have.string('This account already exists');
         expect(body).to.have.string('Sign in');
         expect(body).to.have.string('Reset password');
-        expect(body).to.have.string('This link is only valid for 30 minutes.');
+        expect(body).to.have.string('This link is only valid for 60 minutes.');
```
Comment (ob6160) [inline cypress/integration/ete/registration/register.cy.ts:212]: following a discussion yesterday we decided to drop the "only" from the message copy now the time is longer than 30 mins, so it should now read:

"This link is valid for 60 minutes."

**Review by ob6160 — CHANGES_REQUESTED** [review]

File: cypress/integration/ete/registration/register.cy.ts (hunk 2 above)
Comment (pvighi) [reply cypress/integration/ete/registration/register.cy.ts:212]: I'm not sure what is so wrong about saying that it's "only" valid for 60 minutes that would justify changing it everywhere but if the decision is already made I'll change it

File: src/client/pages/ResetPasswordSessionExpiredPage.tsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ export const ResetPasswordSessionExpiredPage = () => {
       formPageTrackingName="reset-password-session-expired"
     >
       <MainBodyText>
-        The link we sent you was valid for 30 minutes and it has now expired.
+        The link we sent you was valid for 60 minutes and it has now expired.
```
Comment (pvighi) [inline src/client/pages/ResetPasswordSessionExpiredPage.tsx:35]: a lot (maybe all) of these `*SessionExpiredPage` pages don't seem to be being used currently. I could test my change by going to the url manually but when the token actually is invalid it just stays in the regular page (password reset in this case) and display a different "session expired" message

File: cypress/integration/ete/registration/register.cy.ts (hunk 2 above)
Comment (pvighi) [reply cypress/integration/ete/registration/register.cy.ts]: I'm not really sure how the "changes requested" feature works...
I made the required changes, do you have to click something to make it official that your requested changes were made ?

Comment (ob6160) [reply cypress/integration/ete/registration/register.cy.ts]: you can re-request a review, or like i'm about to do i can just put a new review through myself 👍

**Review by ob6160 — APPROVED** [review]
⏳ 🔢 📧

**Pull request merged**